tests/test_sign_edge.py

```diff
@@ -0,0 +1,43 @@
+from faker import Faker
+from hathorlib.scripts import P2PKH
+from hathorlib.utils import get_address_from_public_key_hash, get_hash160
+
+from app_client.transaction import ChangeInfo, TxInput, TxOutput
+from utils import fake_path, fake_tx
+
+fake = Faker()
+
+
+def test_last_chunk_only_change(cmd, public_key_bytes):
```

Comment:
public_key_bytes is a list of public keys derived from the speculos key at `m/44'/280'/0'/0/index`
This is to create actual scripts, that can be validated as change outputs by the device.

Reply:
question(non-blocking): Why not add this comment to the code directly as a docstring? It seems important for future maintenance.

Reply:
This is part of the pytest injection system, its configured on the conftest.py file and there we have an explanation for this.
Also, the first thing on the file is the speculos default seed for testing.